apps/desktop/src/renderer/pages/settings/sections/Section.module.css

```
@@ -621,11 +621,40 @@
+  background: color-mix(in srgb, var(--warning, #f59e0b) 10%, transparent);
+  color: var(--warning, #f59e0b);
+  font-size: 0.8125rem;
+  font-weight: 500;
+}
+
+/* Spinner animation for loading states */
+.pluginSpinner {
+  animation: pluginSpin 1s linear infinite;
+}
+
+@keyframes pluginSpin {
```

Stylelint (17.7.0)

[error] Expected keyframe name "pluginSpin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)